experts-migration-commands.php

```diff
@@ -0,0 +1,742 @@
+		<p>Tel:  604.802.0779</p>
+		<!-- /wp:paragraph -->
+		
+		<!-- wp:paragraph -->
+		<p>Email: <a href="mailto:collins.maina@ubc.ca">collins.maina@ubc.ca</a></p>
+		<!-- /wp:paragraph -->';
+
+		$erik_rolfsen['title'] = 'Erik Rolfsen Media Relations';
+		$erik_rolfsen['content'] = '<!-- wp:heading {"level":3} -->
+		<h3>Media Contact</h3>
+		<!-- /wp:heading -->
+		
+		<!-- wp:paragraph -->
+		<p>Erik Rolfsen</p>
+		<!-- /wp:paragraph -->
+		
+		<!-- wp:paragraph -->
+		<p>Media Relations Specialist</p>
+		<!-- /wp:paragraph -->
+		
+		<!-- wp:paragraph -->
+		<p>Tel: 604.822.2644</p>
+		<!-- /wp:paragraph -->
+		
+		<!-- wp:paragraph -->
+		<p>Cell: 604.209.3048</p>
+		<!-- /wp:paragraph -->
+		
+		<!-- wp:paragraph -->
+		<p>Email: <a href="mailto:erik.rolfsen@ubc.ca">erik.rolfsen@ubc.ca</a></p>
+		<!-- /wp:paragraph -->';
+
+
+		$collins_post = array(
+			'post_title'	=> $collins_maina['title'],
+			'post_content'  => $collins_maina['content'],
+			'post_status'   => 'publish',
+			'post_type'		=> 'wp_block'
+
+		);
+		$collins_maina['id'] = wp_insert_post( $collins_post, false );
+
+		$alex_post = array(
+			'post_title'	=> $alex_walls['title'],
+			'post_content'  => $alex_walls['content'],
+			'post_status'   => 'publish',
+			'post_type'		=> 'wp_block'
+
+		);
+		$alex_walls['id'] = wp_insert_post( $alex_post, false );
+
+		$lou_post = array(
+			'post_title'	=> $lou_corpuz_bosshart['title'],
+			'post_content'  => $lou_corpuz_bosshart['content'],
+			'post_status'   => 'publish',
+			'post_type'		=> 'wp_block'
+
+		);
+		$lou_corpuz_bosshart['id'] = wp_insert_post( $lou_post, false );
+
+		$patty_post = array(
+			'post_title'	=> $patty_wellborn['title'],
+			'post_content'  => $patty_wellborn['content'],
+			'post_status'   => 'publish',
+			'post_type'		=> 'wp_block'
+
+		);
+		$patty_wellborn['id'] = wp_insert_post( $patty_post, false );
+
+		$erik_post = array(
+			'post_title'	=> $erik_rolfsen['title'],
+			'post_content'  => $erik_rolfsen['content'],
+			'post_status'   => 'publish',
+			'post_type'		=> 'wp_block'
+
+		);
+		$erik_rolfsen['id'] = wp_insert_post( $erik_post, false );
+
+		$this->media_contacts = array(	
+			// 'Erik Rolfsen' 
+			'arts' 						=> $erik_rolfsen['id'],	
+			'medicine' 					=> $erik_rolfsen['id'],	
+			'library' 					=> $erik_rolfsen['id'],
+			// 'Lou Corpuz-Bosshart'	
+			'applied_science' 			=> $lou_corpuz_bosshart['id'],	
+			'forestry' 					=> $lou_corpuz_bosshart['id'],	
+			'pharmaceutical_sciences' 	=> $lou_corpuz_bosshart['id'],
+			// 'Alex Walls'
+			'law' 						=> $alex_walls['id'],	
+			'science' 					=> $alex_walls['id'],
+			// Collins Maina	
+			'education' 				=> $collins_maina['id'],	
+			'land_and_food_systems' 	=> $collins_maina['id'],	
+			'sauder_school_of_business' => $collins_maina['id'],	
+			// Patty Wellborn
+			'okanagan'					=> $patty_wellborn['id']
+		);
+
+	}
+	protected function process_social_media( $social_media ){
+		$social_media_options = array(
```

Comment:
Any chance we can use HTTPS urls by default? I'm not sure I 100% understand the data here, but if we can, it'll help save one redirect.